### PR TITLE
3 create homepage skeleton

### LINKED ISSUE #3: Create homepage skeleton

### PR DESCRIPTION
**Pull request by HaashimB**
Closes #3 

- Added Semantic-UI to project

- Built basic layout of the page. Also changed Eslint from yml to js. 

- Added placeholder background image and Nav Bar

### CODE REVIEW

**Comment by HaashimB** [comment]
Screenshot of homepage: https://imgur.com/jrfsck1

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import 'semantic-ui-css/semantic.min.css';
 
 import './sass/index.sass';
 import Router from './Router';
 
+
```
Comment (salahusama) [inline src/index.js]: Nit: Extra new line

File: src/sass/App.sass [hunk 2]
```diff
@@ -1,33 +1,37 @@
 .App
-  text-align: center
+  background-image: url(../assets/prayer.jpg)
```
Comment (salahusama) [inline src/sass/App.sass]: I'm not sure what effect we're going with, but I don't think this is what we want. If I'm not mistaken, this way the background of the web page will always be a static image. When we scroll the page, the image should move up as well. This means its not a background but an image on the page _(in a div or sth)_.

File: src/sass/App.sass [hunk 3]
```diff
@@ -1,33 +1,37 @@
...
-  
-
+  position: fixed
+  top: 0
+  width: 100%
+  border-bottom: 1px solid #0000001f
+  background-color: #ffffff66
+  height: 45px
+  justify-content: flex-end
+  .HeaderLeft
```
Comment (salahusama) [inline src/sass/App.sass]: It would be better to add a new line before this.

File: src/sass/App.sass [hunk 4]
```diff
@@ -1,33 +1,37 @@
...
+    display: flex
+    color: black
+    width: 100%
+    justify-content: flex-start
+    align-items: center
+    margin: 0 0 0 10px
+    .HeaderTitle
```
Comment (salahusama) [inline src/sass/App.sass]: It would be better to add a new line before this.

File: src/sass/App.sass [hunk 5]
```diff
@@ -1,33 +1,37 @@
...
+      margin: 0 0 0 10px
+  .HeaderRight
```
Comment (salahusama) [inline src/sass/App.sass]: It would be better to add a new line before this.

File: src/sass/App.sass [hunk 6]
```diff
@@ -1,33 +1,37 @@
...
+    width: 410px
+    height: 50px
+    display: flex
+    justify-content: space-evenly
+    align-items: center
+    .HeaderHome,
```
Comment (salahusama) [inline src/sass/App.sass]: It would be better to add a new line before this.

File: src/sass/App.sass [hunk 7]
```diff
@@ -1,33 +1,37 @@
...
+    .HeaderArticles,
+    .HeaderContact
```
Comment (salahusama) [inline src/sass/App.sass]: I think it would be better to have the 3 on the same line.

File: src/sass/App.sass [hunk 8]
```diff
@@ -1,33 +1,37 @@
...
+      margin: 0
+      cursor: pointer
+    .HeaderHome:hover,
```
Comment (salahusama) [inline src/sass/App.sass]: It would be better to add a new line before this.

File: src/sass/App.sass [hunk 9]
```diff
@@ -1,33 +1,37 @@
...
+    .HeaderArticles:hover,
+    .HeaderContact:hover
```
Comment (salahusama) [inline src/sass/App.sass]: I think it would be better to have the 3 on the same line.

**Comment by salahusama** [comment]
You can ignore the circleci build error.

Also, you can attach an image directly in github by just copying the image itself and pasting it in the comment.

image

File: src/components/homepage/App.js [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+import React from 'react';
+import Header from './Header';
+import Background from './Background';
+
+class App extends React.Component {
+  render(){
+    return(
+      <div className="App">
+        <div className="Background">
+          {Background}
```
Comment (salahusama) [inline src/components/homepage/App.js]: Would `coverImg` or sth be a more appropriate name so we don't get confused later?

**Comment by salahusama** [comment]
Any updates on this, @HaashimB?

**Comment by HaashimB** [comment]
@salahusama can you email me an mai youth logo with no background and I'll add it in, inshaAllah

**Comment by salahusama** [comment]
I have no idea why the tests are failing so I'll ignore it... Will check it later.

**Pull request merged**